gemfileparser/__init__.py

```diff
@@ -92,27 +80,33 @@ def __init__(self, filepath, appname=''):
         self.dependencies = {
             'development': [],
             'runtime': [],
+            'dependency': [],
             'test': [],
             'production': [],
             'metrics': []
         }
         self.contents = self.gemfile.readlines()
-        if filepath.endswith('gemspec'):
+        path = ('gemspec', 'podspec')
+        if filepath.endswith(path):
             self.gemspec = True
         else:
             self.gemspec = False
 
     @staticmethod
     def preprocess(line):
-        '''Removes the comment portion and excess spaces.'''
+        """
+        Return line after removing comment portion and excess spaces.
+        """
 
-        if "#" in line:
+        if '#' in line:
             line = line[:line.index('#')]
         line = line.strip()
         return line
 
     def parse_line(self, line):
-        '''Parses each line and creates dependency objects accordingly'''
+        """
+        Parses each line and creates dependency objects accordingly.
```

Comment:
Please use the imperative style for docstrings.

Reply:
I will change it soon :)